common-tools/clas-tracking/src/main/java/org/jlab/clas/tracking/kalmanfilter/helical/MeasVecs.java

```diff
@@ -71,6 +71,9 @@ public double[] H(AStateVecs.StateVec stateVec, AStateVecs sv, MeasVec mv, Swim
                 SVplus.toDoca();
                 SVminus.toDoca();
             }
+            
+            SVplus.updateFromHelix();
```

Comment:
the update of the state vector from the helix parameters is already done in rollBack and toDoca and I think the two lines here could be removed